package com.Exception;

import java.util.Scanner;

public class Try_Catch_finalBlock_Exception {
	public static void main(String[] args) {
		
	
	try
	{
		System.out.println("like share");
		int a = 20, b=0, c;
		c= a/b;
		System.out.println(c);
			}
	catch(Exception e)
	{
		System.out.println("not subscribe");
		System.out.println("watch a full video.....");
	}
	finally{
		
		System.out.println("thank yu");
	}
}
	}

